src/addons/transitions/ReactCSSTransitionGroupChild.js

```diff
@@ -29,6 +29,8 @@ var NO_EVENT_TIMEOUT = 5000;
 
 var noEventListener = null;
 
+var REQUEST_ANIMATION_FRAME = (window && window.requestAnimationFrame) ? window.requestAnimationFrame : false;
```

Comment:
We have a `requestAnimationFrame` module which we should probably use instead (that falls back to a setTimeout though, so you might just want `nativeRequestAnimationFrame`)